### DISCUSSION TITLE
Precise control over zoom?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#3834] [Q&A]
I am playing around with the new 1.1.1 version on Windows.

The action of the zoom control at the bottom right of the page seems to have changed slightly (the values that it takes on when dragged aren't the same as with the 1.1.0 version).

I was wondering if there is any way to specify an exact value for this control - maybe a dialog to enter the zoom value? I would find this very useful, but I can't find any such control. Am I missing it somewhere?

**Comment by bhennion** [comment]
I don't think there is any feature like that. You can make a feature request, for instance to make the zoom percentage display editable (like in many other apps).

In the meantime, the most precise zoom level control I have is with `Ctrl + MouseWheel`. It gives you ~6% steps.

**Reply by NelkonParker** [reply]
Thanks for that. However, Ctrl+Mousewheel does nothing very useful for me (on Windows 10 with 1.1.1). I get e.g. 132% -> 192% -> 284%.

I'll raise a feature request at some point in the near future.
